### ISSUE TITLE
課題の進め方について

### ISSUE THREAD

**Issue by phb2** [hikky-coop/rpg#12]
課題の進め方について。

**Comment by phb2** [comment]
どれから先にやればいいのか。

* プル・リクエスト対応 #2 
* ソースコードへのコメント追記/整理 #10 
* 仕様の文書化 #11 #7 

があるので、先に仕様の文書化をやってしまって、読んでもらってる間にプル・リクエスト対応がいいかな。

**Comment by phb2** [comment]
いや、プル・リクエスト対応が先かな。

**Comment by phb2** [comment]
現在は仕様書書いてます。ちょっと時間かかりそう。

**Comment by u-Bano** [comment]
ひきねこさん乙です

**Comment by phb2** [comment]
おつです。ここは雑談スレにしちゃいますね。

**Comment by phb2** [comment]
GitHubの組織垢の扱いに慣れていないのでコメントへのレスなど取りこぼしがあるかもしれないです。慣れてくればレスポンスが早くなると思います。

**Comment by phb2** [comment]
ここ数日で課題ごとのコメントに対するレスの取りこぼしが（主に私に）見られるようです。
Open/Closedな課題は現在１２ありますが、少しだけコメントを追うのがキツくなってきました。
とりあえず現状報告まで。

**Comment by phb2** [comment]
雑談スレで大まかな方針を話し合いで決めて、それに沿って各課題を処理するというのはどうでしょう？
とりあえず提案まで。

**Comment by u-Bano** [comment]
> 雑談スレで大まかな方針を話し合いで決めて、それに沿って各課題を処理するというのはどうでしょう？

同意です、それでよいと思います

**Comment by phb2** [comment]
＞既に存在するIssueについて話し合うときはこの雑談スレから誘導して行うというのも手かもしれませんね
同意です。それが楽そうです。
個人的には共同作業のノウハウを目に見える形でwikiなどに残したいと思っているのですが、それについてはどうでしょうか？
今回の課題の件も文書化したいと思うのですが。

**Comment by phb2** [comment]
それからIssueについてのアイデアを思いついたのでメモします。
u-Bano氏との話し合いで「雑談スレ→他Issue」という作業フローが生まれましたが、これをhikky-coopと他のプロジェクトにも適用してみるアイデアです。
「話し合いの場」が今より明確になるので、迷子になることが少なくなるような気がします。構造的には木構造ですね。

```
hikky-coopの雑談スレ -> rpgの雑談スレ -> rpgのIssue#1
                                      -> rpgのIssue#2
                     -> go-2chの雑談スレ -> go-2chのIssue#1
                                         -> go-2chのIssue#2
```

**Comment by u-Bano** [comment]
> 個人的には共同作業のノウハウを目に見える形でwikiなどに残したいと思っているのですが、それについてはどうでしょうか？
> 今回の課題の件も文書化したいと思うのですが。

今後新しく加わる人がいれば参考になると思いますし良いアイデアだと思います

**Comment by u-Bano** [comment]
> 「雑談スレ→他Issue」という作業フローが生まれましたが、これをhikky-coopと他のプロジェクトにも適用してみるアイデアです。
> 「話し合いの場」が今より明確になるので、迷子になることが少なくなるような気がします。構造的には木構造ですね。
> 上記は思い付きなので、他に定石などがあればそれを採用するのがいいと思われます。

よいアイデアだと思います
まだ始まったばかりですしこれからまだまだ色々と模索していくことになりそうですね

**Comment by phb2** [comment]
そうですね。アイデアなどは雑談スレに書いておけば見直せますね。ジャンル的には開発体制や組織体制の構築になるのかな？なかなか一人でやってると未知の分野です。
分業についてなんですが、どうしましょう。試しにコメントの追記Issueについて話し合ってみますか？

**Comment by u-Bano** [comment]
そうですね、お試しでコメントの追記について当該Issueでやってみましょう
